platform/integrations/karpenter.mdx

```diff
@@ -0,0 +1,449 @@
+---
+title: Using Karpenter with vCluster Platform
+sidebar_label: Karpenter
+sidebar_position: 7
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Flow, { Step } from "@site/src/components/Flow";
+import Label from "@site/src/components/Label";
+
+vCluster is compatible with Karpenter and vCluster's features can be used in conjuntion with Karpenter to
+further reduce infrastructure costs.
+
+This article will explain how to [Setup Designated Node Pool for a vCluster](#setup-designated-node-pool-for-a-vcluster), [Setup Designated Node
+Pool for a Project](#setup-designated-node-pool-for-a-project), and [Setup Designated Node Pool for Dev/Prod Workloads Accross All vClusters](#setup-designated-node-pool-for-a-project). All
+three of which can be powerful cost saving tools.
+
+This article assumes you are using a host cluster provisioned on one of the following managed Kubernetes
+services: EKS (Elastic Kubernetes Service) or AKS (Azure Kubernetes Service).
+
+Much of this article borrows and builds upon the [Getting Started with Karpenter](https://karpenter.sh/docs/getting-started/getting-started-with-karpenter/) documentation and the [Karpenter Provider Azure README.md](https://github.com/Azure/karpenter-provider-azure).
+
+### Pre-requisites for Setups
+1. A kubernetes cluster setup in either AKS or EKS.
+2. Necessary permissions in chosen cloud provider to use Karpenter.
```

Comment:
Do we know what it means by necessary permissions? This is very vague.

Reply:
@deniseschannon this can probably be reword. The permissions for cloud provider vary and have more to do with provisioning. If you already have a cluster then you already know what they are so we don't need to outline them. What I can explain is what vcluster platform and k8s permissions you'd need.

Reply:
I'd assume the cluster is already provisioned, so what permissions are required for vCluster Platform/Karpenter. 

Is there a way to check if those permissions exists. I'd add those as well

Reply:
Updated.